ansible_collections/arista/avd/execution-environment.yml

```diff
@@ -0,0 +1,12 @@
+# execution-environment.yml
```

Comment:
IMO we should keep this file out of the collection package. So move it to a folder under the root.

Reply:
Root of the collection is standard place for this file meaning you don't have to provide filename using `-f` knob. It is also used with `ansible-builder introspect --sanitize`.

If we moved it in other location, it will make things more complex